pytx3/pytx3/cli/fetch2.py

```diff
@@ -56,3 +59,45 @@ def __init__(
         self.owner = owner
         self.threat_types = threat_types
         self.additional_tags = additional_tags
+
+    def execute(self, dataset: Dataset) -> None:
+        self._fetch_threat_updates(
+            dataset.config.privacy_groups[0],
+            self.start_time,
+            self.stop_time,
+            self.owner,
+            self.threat_types,
+            self.additional_tags
+        )
+
+    def _fetch_threat_updates(
+        self,
+        privacy_group: int,
```

Comment:
Eventually, we're going to want to change this to a collaboration object right?

Reply:
yup